packages/remix-dev/vite/plugin.ts

```diff
@@ -928,7 +929,6 @@ export const remixVitePlugin: RemixVitePlugin = (options = {}) => {
           parserOpts: {
             sourceType: "module",
             allowAwaitOutsideFunction: true,
-            plugins: ["jsx", "typescript"],
```

Comment:
I think we still want this change. Can we reopen this PR and scope it down to just this one liner?

Reply:
Thanks for re-opening! I updated a PR to include only this change.

Btw, recently macos/webkit CI is often failing with `vite-dotenv-test.ts` but I'm not sure if it's related to my PR. Is this something I should worry about?

Reply:
@hi-ogawa I'm looking into that test failure now.

Reply:
This PR got the test passing: https://github.com/remix-run/remix/pull/8177. I've just merged dev into your PR.